File: 2023/UICPC Regional/Round #12/solutions/compass/per.js
// I have no idea what I'm doing.

var n1 = parseInt(readline());
var n2 = parseInt(readline());
n2 -= n1;
if (n2 > 180) {
    n2 -= 360;
}
if (n2 <= -180) {
    n2 += 360;
}
print(n2);

File: 2023/UICPC Regional/Round #12/solutions/compass/per.go
// I have no idea what I'm doing.

package main

import (
    "fmt"
)

func main() {
    var n1, n2 int64

    fmt.Scanf("%d\n%d", &n1, &n2)
    n2 -= n1
    if n2 > 180 {
       n2 -= 360
    }
    if n2 <= -180 {
       n2 += 360
    }
    fmt.Printf("%d\n", n2)
}
